modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -3448,7 +3448,7 @@ void GDScriptAnalyzer::reduce_call(GDScriptParser::CallNode *p_call, bool p_is_a
 			String base_name = is_self && !p_call->is_super ? "self" : base_type.to_string();
 #ifdef SUGGEST_GODOT4_RENAMES
 			String rename_hint;
-			if (GLOBAL_GET(GDScriptWarning::get_settings_path_from_code(GDScriptWarning::RENAMED_IN_GODOT_4_HINT)).booleanize()) {
+			if (GLOBAL_GET("debug/gdscript/warnings/renamed_in_godot_4_hint").booleanize()) {
```

Comment:
I guess the call to `.booleanize()` is no longer needed now that the type is `bool`?

Reply:
Probably yes, since there is `Variant::operator bool()`. But this means it was not necessary before either.